Content.Shared/Chemistry/Prototypes/InjectorModePrototype.cs

```diff
@@ -0,0 +1,158 @@
+﻿using Content.Shared.Chemistry.Components;
+using Content.Shared.FixedPoint;
+using Robust.Shared.Audio;
+using Robust.Shared.Prototypes;
+using Robust.Shared.Serialization;
+
+namespace Content.Shared.Chemistry.Prototypes;
+
+[Prototype("InjectorMode"), Serializable]
+public sealed partial class InjectorModePrototype : IPrototype
```

Comment:
In order for your prototype to posses the `Abstract` and `Parents` it needs to also inherit `IInheritingPrototype`, then implement those fields.

```suggestion
public sealed partial class InjectorModePrototype : IPrototype, IInheritingPrototype
{
    /// <inheritdoc/>
    [IdDataField]
    public string ID { get; private set; } = default!;

    /// <inheritdoc/>
    [ParentDataField(typeof(AbstractPrototypeIdArraySerializer<InjectorModePrototype>))]
    public string[]? Parents { get; private set; }

    /// <inheritdoc/>
    [AbstractDataField, NeverPushInheritance]
    public bool Abstract { get; private set; }
```